src/index.js

```diff
@@ -47,7 +47,7 @@ class Dotenv {
     }
 
     Object.keys(blueprint).map(key => {
-      const value = env[key]
+      const value = env[key] || vars[key]
```

Comment:
Can you verify that? I think dotenv variables are always strings and a string with a 0 doesn't return falsy.

Try this:
'''
console.log("0" || "we got a problem")
'''